src/gadgets/util.rs

```diff
@@ -203,6 +203,34 @@ impl<F: PrimeField, const T: usize> MainGate<F, T> {
         Ok(out)
     }
 
+    /// Add `lhs` assigned value to `rhs` constant
+    ///
+    /// By one row with simple expression
+    /// ```markdown
+    /// input * q_i + output * q_o + rc  = 0
+    /// lhs   * 1   + sum    * -1  + rhs = 0
+    /// ```
+    pub fn add_with_const(
```

Comment:
We can also use `apply` instead of adding a new function

Reply:
I'll be honest - I reread the `apply` logic every time and this API has never once become quickly clear to me, or even more intuitive. 

I.e. it is a very mathematical function, but it is a pain to use it in the final code.

I just don't want to spend time now on such a voluminous change as refactoring it, but it definitely needs to be improved in the future. So right now I just added the function I need.

Reply:
In my case, when I use it often, I will be able to apply it quickly. After long time, I need read its code to pick up the memory.